src/runtime_src/core/include/xrt_mem.h

```diff
@@ -30,14 +30,14 @@ extern "C" {
  * bits 24 ~ 31: BO flags
  */
 
-#define XRT_BO_FLAGS_MEMIDX_MASK	(0xFFFFFFUL)
-#define	XCL_BO_FLAGS_NONE		(0)
-#define	XCL_BO_FLAGS_CACHEABLE		(1 << 24)
-#define	XCL_BO_FLAGS_SVM		(1 << 27)
-#define	XCL_BO_FLAGS_DEV_ONLY		(1 << 28)
-#define	XCL_BO_FLAGS_HOST_ONLY		(1 << 29)
-#define	XCL_BO_FLAGS_P2P		(1 << 30)
-#define	XCL_BO_FLAGS_EXECBUF		(1 << 31)
+const uint32_t 	XRT_BO_FLAGS_MEMIDX_MASK   = 0xFFFFFF;
```

Comment:
Please include <stdint.h> for C and <cstdint> for __cplusplus;  the header file should be standalone compilable.